docs/StardustDocs/topics/setup/SetupKotlinNotebook.md

```diff
@@ -71,6 +71,13 @@ You can explicitly define the version you want:
 Or use the latest stable version of Kotlin DataFrame
 (specified in [Kotlin Jupyter descriptors](https://github.com/Kotlin/kotlin-jupyter-libraries)):
 
+<warning>
+For version `1.0.0-Beta3`, in notebooks use version `1.0.0-Beta3n` instead.
```

Comment:
is it possible to make it a template with dfVersion?

Reply:
well hopefully we don't need that here. This is only a temporary thing that hopefeully only applies for Beta3